core-processor/src/processor.rs

```diff
@@ -107,6 +108,14 @@ pub fn process_many<A: ProcessorExt + EnvExt + Into<ExtInfo> + 'static, E: Envir
     journal
 }
 
+fn is_non_executable(actor: Option<&ExecutableActor>, dispatch: &Dispatch) -> Option<ExitCode> {
```

Comment:
The function name doesn't reflect the return value. Usually `is_something()` returns `bool`.

Reply:
maybe `check_is_non_executable(...) -> Result<ExitCode, ()>`?